userguide/directoryservices.rst

```diff
@@ -305,21 +313,25 @@ Once populated, the AD users and groups will be available in the
 drop-down menus of the :guilabel:`Permissions` screen of a dataset.
 
 The Active Directory users and groups that are imported to the %brand%
-system are shown by typing commands in the %brand% :ref:`Shell`:
+system are shown by typing commands in the %brand% :ref:`shell`:
```

Comment:
This was upper case to show that it was a link to the Shell section rather than a generic shell.  But I guess it's not required.